pkg/setting/setting.go

```diff
@@ -14,6 +14,9 @@ import (
 
 	"github.com/Unknwon/com"
 	"github.com/macaron-contrib/session"
+	_ "github.com/macaron-contrib/session/mysql"
+	_ "github.com/macaron-contrib/session/postgres"
+	_ "github.com/macaron-contrib/session/redis"
```

Comment:
Pretty confident that redis/mysql wouldn't even have worked without importing these drivers.